beetsplug/export.py

```diff
@@ -52,6 +54,23 @@ def __init__(self):
                     'sort_keys': True
                 }
             },
+            'csv': {
+                # csv module formatting options
+                'formatting': {
+                    'delimiter': ',', # column seperator
+                    'dialect': 'excel', # the name of the dialect to use
+                    'quotechar': '|'
```

Comment:
I think you probably want `quotechar` to default to "? In fact, it doesn't seem like the most useful thing to want to override… maybe we can get away without this option.

Reply:
Sounds good, I'll remove it